src/ui/app/jsx/active-compactions.jsx

```diff
@@ -48,9 +48,12 @@ const ActiveCompactions = CreateReactClass({
         dataType: 'json',
         complete: function(data) {
           try {
-            this.component.setState({activeCompactions: data.responseJSON});
+            this.component.setState({
+                pendingCompactions: data.responseJSON.pendingCompactions,
+                activeCompactions: data.responseJSON.activeCompactions
+            });
           } catch(error) {
-            this.component.setState({activeCompactions: []});
+            this.component.setState({pendingCompactions:-1, activeCompactions: []});
```

Comment:
It might be good to add some tracing here: `console.error('Error occurred while retrieving compaction stats:', error);`
wdyt?

Reply:
Sure thing, added the trace.